### PR TITLE
PR#4233-Fix broken example

### PR DESCRIPTION
**Pull request by ebonow**
Based on PR from @cseas https://github.com/JedWatson/react-select/pull/4233 🙏 

The work done is to correct a typo that prevents a code example from working properly.

### CODE REVIEW

**Comment by ebonow** [comment]
All tests passing, pending peer review.

**Review by kylemh — APPROVED** [review]

**Comment by ebonow** [comment]
@bladey This requires approval from a reviewer with write access

**Review by Methuselah96 — APPROVED** [review]
Since this is just fixing something in the `docs` and the fix is clearly correct, I'm gonna go ahead and merge this.

**Pull request merged**
